framework/captcha/CaptchaAsset.php

```diff
@@ -8,7 +8,7 @@
 namespace yii\captcha;
```

Comment:
Captcha should be moved to separate repository. Script should be adjusted not to use jQuery.

Could be done in separate pull request after this one.